src/appkit.cs

```diff
@@ -716,6 +716,11 @@ interface NSApplication : NSAccessibilityElementProtocol, NSAccessibility {
 		[Lion, Field ("NSApplicationLaunchRemoteNotificationKey")]
 		NSString LaunchRemoteNotificationKey { get; }
 
+#if !XAMCORE_4_0
```

Comment:
The !XAMCORE_4_0 is supposed to be on the old binding in NSRemoteNotifications_NSApplication